### ISSUE TITLE
Bump relayer fee in UI 

### ISSUE THREAD

**Issue by alexwhte** [connext/monorepo#3011]
## Problem
If transactions fail due to low relayer fee, we need a way to tell them and unblock their transfer

## Proposed solution
SEE SPEC
https://www.notion.so/connext/Bump-slippage-Relayer-fee-UX-bd1e267baca046988976a64879224abf

## Acceptance Criteria
- [ ]  User is notified of the failure
- [ ]  User is presented option to increase the fee to amount we provide 
- [ ]  User can increase it
- [ ]  Execute retries automatically (if they bump OR not)

**Comment by sanchaymittal** [comment]
<img width="445" alt="Screenshot 2023-01-17 at 5 40 20 PM" src="https://user-images.githubusercontent.com/31698676/212895601-fa00b8d9-6e90-462c-a539-efde134131e3.png">

seems like it's private page

**Comment by rhlsthrm** [comment]
#3108 will address the first part of this!

**Comment by alexwhte** [comment]
oops i tested this early but saw an error #3137 . Feel free to close that if you are aware already/working on it

**Comment by alexwhte** [comment]
Thanks @pjenviri !!
https://www.awesomescreenshot.com/video/14151973?key=759a67ee44dc7e78f6c97b5a1b06814e

A few final updates from this
1. Show the error message immediately to the user upon entering the transfer page. So they don't need to figure out that they need to click the red text 
2. Once the 'Update in progress...' status completes, switch to a completed state
a. Say 'Relayer fee updated' in the button and turn the button green
b. Update the Relayer fee shown in xcall (left side) to reflect the new amount 
3. Once the transfer completes, switch the 'Action required' status to a green color with the green checkmark
This way users and support teams can keep track of what was fixed vs. what actually needs action 
https://amarok.connextscan.io/tx/0x1093b67015a6345e8c0e08d0106a52bd7ce9db8e7a47773ab2d3d75f94ac846b

**Comment by alexwhte** [comment]
number 2 looks good

but 1 and 3 still not as expected
https://www.awesomescreenshot.com/video/14216593?key=e694d832c113a3f7fba66342b1ce093c

**Comment by alexwhte** [comment]
from user and router feedback: 
This is too hidden in Explorer. Also, it still shows processing in the loading animation

**Changes.** 
1. Clicking the tile in 'Latest Transfers' presents the same 'bumping' popup to the user that we now have in their individual Transfers page
2. The clock animation should switch to a failure state  
![image](https://user-images.githubusercontent.com/37959170/214209223-4425ea26-bfc7-4eb9-87bd-0064f134c24d.png)
Action Required: Destination gas fee insufficient
The destination gas paid to complete this trade is currently too low. 
You may bump the amount of gas below for the relayer to execute your transfer.  If you take no action, the transfer will continue to retry with the current amount. 

For #2, we will probably want to test this in tandem with #3115. You can leave this out of scope here in order to implement and test any new animation all together.

**Comment by pjenviri** [comment]
> number 2 looks good
> 
> but 1 and 3 still not as expected
> https://www.awesomescreenshot.com/video/14216593?key=e694d832c113a3f7fba66342b1ce093c

For 1; the modal immediately appears when opens the transfer page.
For 3; we're not sure how to handle it because there's no status between updating the relayer and completed. The current UI displays the status from the field: `error_status`. If the `status` is completed, it will then show success.

**Comment by rhlsthrm** [comment]
> For 3; we're not sure how to handle it because there's no status between updating the relayer and completed. The current UI displays the status from the field: error_status. If the status is completed, it will then show success.

`error_status` should be updating properly when the issue is resolved. Let us know if it's not.

**Comment by alexwhte** [comment]
1 looks great
for 3 reminder to self to  verify with this transfer i just bumped https://amarok.connextscan.io/tx/0x43a18ae34446cdb383951010cfda1caedce9bdb1b42efda81a6394603df5c4a1
